### PR TITLE
Remote project fixes

### PR DESCRIPTION
**Pull request by AMZN-Phil**
Fix objects not being downloaded to the chosen path.
Fix remote projects not being shown if there are no local projects.
Adding placeholder text and warning icon in add remote project dialog, 
Fix exception using logger in github_utils.py

Signed-off-by: AMZN-Phil <pconroy@amazon.com>

### CODE REVIEW

File: Code/Tools/ProjectManager/Source/AddRemoteProjectDialog.cpp [hunk 1]
```diff
@@ -48,16 +48,30 @@ namespace O3DE::ProjectManager
 
         m_repoPath = new FormLineEditWidget(tr("Remote URL"), "", this);
         m_repoPath->setMinimumSize(QSize(600, 0));
+        m_repoPath->lineEdit()->setPlaceholderText("http://github.com/example.git");
```
Comment (AMZN-alexpete) [inline Code/Tools/ProjectManager/Source/AddRemoteProjectDialog.cpp:51]:
```suggestion
        m_repoPath->lineEdit()->setPlaceholderText("http://github.com/o3de/example.git");
```
repository urls generally have an owner followed by the repo.git

File: Code/Tools/ProjectManager/Source/AddRemoteProjectDialog.cpp [hunk 2]
```diff
@@ -48,16 +48,30 @@ namespace O3DE::ProjectManager
...
         vLayout->addWidget(m_repoPath);
 
         vLayout->addSpacing(10);
 
+        QHBoxLayout* warningHLayout = new QHBoxLayout(this);
+
+        QLabel* warningIcon = new QLabel(this);
+        //warningIcon->setObjectName("projectWarningIconOverlay");
```
Comment (AMZN-alexpete) [inline Code/Tools/ProjectManager/Source/AddRemoteProjectDialog.cpp:59]:
```suggestion
```

File: Code/Tools/ProjectManager/Source/AddRemoteProjectDialog.cpp [hunk 3]
```diff
@@ -48,16 +48,30 @@ namespace O3DE::ProjectManager
...
         m_repoPath = new FormLineEditWidget(tr("Remote URL"), "", this);
         m_repoPath->setMinimumSize(QSize(600, 0));
+        m_repoPath->lineEdit()->setPlaceholderText("http://github.com/example.git");
         vLayout->addWidget(m_repoPath);
 
         vLayout->addSpacing(10);
 
+        QHBoxLayout* warningHLayout = new QHBoxLayout(this);
+
+        QLabel* warningIcon = new QLabel(this);
```
Comment (AMZN-alexpete) [inline Code/Tools/ProjectManager/Source/AddRemoteProjectDialog.cpp:58]:
```suggestion
        QHBoxLayout* warningHLayout = new QHBoxLayout();
        QLabel* warningIcon = new QLabel();
```
These should get automatically parented when they are added to layouts

**Review by AMZN-alexpete — APPROVED** [review]
Just a couple minor things and could you add a screenshot of the updated UI with the new warning icon?

**Comment by AMZN-Phil** [comment]
AddProjectWarning

**Review by lumberyard-employee-dm — APPROVED** [review]

**Pull request merged**